pkg/ssh/ssh.go

```diff
@@ -383,15 +386,18 @@ func (l *SSHTunnelList) pickTunnel(addr string) (tunnel, error) {
 		return nil, fmt.Errorf("No SSH tunnels currently open. Were the targets able to accept an ssh-key for user %q?", l.user)
 	}
 	// Prefer same tunnel as kubelet
-	// TODO: Change l.entries to a map of address->tunnel
```

Comment:
As in `map[string]Tunnel`. I think the idea is that this refactor removes `sshTunnelEntry`.

Reply:
thanks for the clarification, I changed the map to `map[string]tunnel` and get rid of `ssh Tunnel Entry`.